lib/trento_web/plugs/charts_disabled_plug.ex

```diff
@@ -0,0 +1,33 @@
+defmodule TrentoWeb.Plugs.ChartsDisabledPlug do
+  @moduledoc """
+    This plug will act as a barrier for the charts endpoint, will return 501 for all the requests.
```

Comment:
I always recommend to write docs in present term `This plug acts as a...`

Reply:
done